src/bfd-disasm.h

```diff
@@ -7,13 +7,13 @@ namespace bpftrace {
 class BfdDisasm : public IDisasm
 {
 public:
-  BfdDisasm(std::string &path);
+  BfdDisasm(std::string &);
   ~BfdDisasm();
 
   AlignState is_aligned(uint64_t offset, uint64_t pc);
 
 private:
-  int fd_ = -1;
+  int fd_;
```

Comment:
Please revert these changes -- no point creating noise in git blame

Reply:
@danobi reverted